docs/src/reference/2_blmo_build.md

```diff
@@ -0,0 +1,24 @@
+# LMO 
+
+This section contains information about interface blmo_interface.jl , build_lmo.jl and time_tracking_lmo.jl
+
+## BLMO_Interface 
+
+```@autodocs
+Modules = [Boscia]
+Pages = ["blmo_interface.jl"]
+```
+
+## Build LMO
```

Comment:
Adapting the BLMO to the node problems

Also move that behind the Time Tracking LMO